hicommonwealth/commonwealth
# 6826 feature flag for growl

## Link to Issue
Closes: #6826 

## Description of Changes
- Added check for the Stake feature flag

## "How We Fixed It"

- Brought in the `useFlag` hook and passed `communityStake` as a param

## Test Plan
- Turn `FLAG_COMMUNITY_STAKE` to `false`
- Ensure the growl does not render
- Turn `FLAG_COMMUNITY_STAKE` to `true`
- Ensure the growl does render
